content/millennium/r4/financial/coverage.md

```diff
@@ -196,7 +211,8 @@ _Implementation Notes_
 
```

Comment:
Should line 206 be updated?

Reply:
Good catch, updated here: https://github.com/cerner/fhir.cerner.com/pull/591/commits/36da5904d52d9c8f8b8c0e73c4437fd4603ce742